typings/index.d.ts

```diff
@@ -378,6 +379,239 @@ declare module 'discord.js' {
 		public once(event: 'end', listener: (collected: Collection<K, V>, reason: string) => void): this;
 	}
 
+	type AllowedImageFormat = 'webp' | 'png' | 'jpg' | 'gif';
+
+	export interface Constants {
+		Package: {
+			name: string;
+			version: string;
+			description: string;
+			author: string;
+			license: string;
+			main: PathLike;
+			types: PathLike;
+			homepage: string;
+			keywords: string[];
+			bugs: { url: string };
+			repository: { type: string, url: string };
+			browser: { [key: string]: boolean };
+			scripts: { [key: string]: string };
+			engines: { [ley: string]: string };
+			dependencies: { [key: string]: string };
+			peerDependencies: { [key: string]: string };
+			devDependencies: { [key: string]: string };
+			[key: string]: any;
+		};
+		browser: boolean;
+		DefaultOptions: ClientOptions;
+		UserAgent: string | null;
+		Endpoints: {
+			botGateway: string;
+			invite: (root: string, code: string) => string;
+			CDN: (root: string) => {
+				Asset: (name: string) => string;
+				DefaultAvatar: (id: string | number) => string;
+				Emoji: (emojiID: string, format: 'png' | 'gif') => string;
+				Avatar: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+				Icon: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+				AppIcon: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+				AppAsset: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+				GDMIcon: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+				Splash: (userID: string | number, hash: string, format: AllowedImageFormat, size: number) => string;
+			};
+		};
+		WSCodes: {
+			1000: 'Connection gracefully closed';
```

Comment:
These are not the actual values, see: https://github.com/discordjs/discord.js/blob/00c4098bb315a92322d708fe4cf71d070508ab69/src/util/Constants.js#L95-L100